internal/model/label.go

```diff
@@ -26,6 +33,12 @@ var BetaLabel Label = Label{
 	URL:         "https://mattermost.com/pl/default-beta-plugins",
 }
 
+var ExperimentalLabel Label = Label{
+	Name:        "Experimental",
+	Description: "This plugin is marked as experimental and not meant for production use. Use with caution!",
```

Comment:
I'm not sure what the tone of urgency should be but 0/5 this seems good to me

```suggestion
	Description: "This plugin is marked as experimental and not meant for production use. Please use with caution.",
```

Reply:
@aaronrothschild What do you think?